blockchain/interface.go

```diff
@@ -22,3 +22,33 @@ type httpClientWrapper interface {
 	PostHTTP(ctx context.Context, endpoint string, data []byte) ([]byte, int, error)
 	IsInterfaceNil() bool
 }
+
+// EndpointProvider is able to return endpoint routes strings
+type EndpointProvider interface {
```

Comment:
maybe remove the Endpoint keyword from methods name. You will anyway call it as endpointProvider.GetVmValuesEndpoint, so it should already makes sense that you are actually geting an endpoint.

Reply:
removed the `Endpoint` suffix from all methods